spec/index.html

```diff
@@ -791,123 +907,705 @@ <h3>Converting a <a>YAML document</a></h3>
+      </p>
+
+      <p>
+        If a cycle is detected,
+        a processing error MUST be returned,
+        and processing aborted. 
       </p>
     </section>
+  </section>
 
-    <section id="convert-scalar">
-      <h3>Converting a <a>YAML scalar</a></h3>
+  <section id="conversion-to-yaml">
+    <h2>Conversion to YAML</h2>
+
+    <p>The conversion process from the <a>internal representation</a>
+      involves turning that representation back into a YAML
+      <a>representation graph</a>
+      and relies on the description of
+      "Serializing the Representation Graph" from the
+      <a data-cite="YAML#dump">3.1.1 Dump</a> section of [[YAML]]
+      for the final serialization.
+    </p>
+
+    <p>
+      As the <a>internal representation</a> is rooted by either
+      an <a>array</a> or a <a>map</a>,
+      the process of transforming the <a>internal representation</a>
+      to YAML begins by preparing an empty <a>representation graph</a>
+      which will be rooted with either a
+      <a>YAML mapping</a> or <a>YAML sequence</a>.
+    </p>
+    <p>
+      Although outside of the scope of this specification,
+      processors MAY use
+      <a>YAML directives</a>, including <a>TAG directives</a>, and
+      <a data-cite="YAML#document-markers">Document markers</a>,
+      as appropriate for best results.
+      Specifically, if the {{JsonLdOptions/extendedYAML}} API flag is `true`,
+      the document SHOULD use the `%YAML` directive with
+      version set to at least `1.2`.
+      To improve readability and reduce document size,
+      the document MAY use a `%TAG` directive appropriate for
+      <a>RDF literals</a> contained within the representation.
+    </p>
+
+    <p class="note">
+      The use of `%TAG` directives in YAML-LD is similar to the use
+      of the `PREFIX` directive in [[?Turtle]]
+       or the general use of terms as prefixes to create
+       <a data-cite="JSON-LD11#dfn-compact-iri">Compact IRIs</a> in [[JSON-LD11]]:
+      they not change  the meaning of the encoded scalars.
+    </p>
+
+    <pre class="example yaml"
+         data-transform="updateExample"
+         data-result-for="YAML-LD with multiple documents"
+         data-content-type="application/ld+json"
+         title="Serialized representation of the extended internal representation">
+    <!--
+    %YAML 1.2
+    %TAG !xsd! http://www.w3.org/2001/XMLSchema%23
+    ---
+    "@context":
+      "@vocab": http://xmlns.com/foaf/0.1/
+    name: !xsd!string Gregg Kellogg
+    homepage: https://greggkellogg.net/
+    depiction: http://www.gravatar.com/avatar/42f948####...####
+    date: !xsd!date "2022-08-08"
+    -->
+    </pre>
+
+    <p class="note">
+      Although allowed within the YAML Grammar, some current YAML parsers
+      do not allow the use of `"#"` within a tag URI &mdash; substituting
+      the `"%23"` escape is a workaround for this problem, that will
+      hopefully become unnecessary as implementations are updated.
```

Comment:
@gkellogg I suggest to collect all these notes under "interoperability consideration", so that implementers can find them all in the same place.